apps/web/test/lib/handleChildrenEventTypes.test.ts

```diff
@@ -96,11 +96,11 @@ describe("handleChildrenEventTypes", () => {
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       // eslint-disable-next-line
-      const { schedulingType, id, teamId, timeZone, users, ...evType } = mockFindFirstEventType({
-        id: 123,
-        metadata: { managedEventConfig: {} },
-        locations: [],
-      });
+      const {schedulingType, id, teamId, timeZone, users, scheduleId, offsetStart, ...evType} = mockFindFirstEventType({
```

Comment:
Prettier won't be happy

```suggestion
      const { schedulingType, id, teamId, timeZone, users, scheduleId, offsetStart, ...evType } = mockFindFirstEventType({
```